### PR TITLE
feat: Added the html elements and login form styles

### PR DESCRIPTION
**Pull request by GusBaamonde**
- The html elements and form styles of the login were added. 
- The spaces will be normalized in another PR.
-  The header will be changed in another PR.

https://cdn.fromdoppler.com/doppler-ui-library/build155/login.html

### CODE REVIEW

File: assets/scss/modules/_forms.scss [hunk 1]
```diff
@@ -54,3 +54,29 @@
...
+  .forgot-link {
+    display: block;
+    margin: $dp-spaces--lv4 $dp-spaces--lv0;
+    color: $dp-color-green;
+    font-size: $dp-font-size-form;
+    font-style: italic;
+    font-family: $dp-font-family;
+
+      &:hover {
+        color: $dp-color-darkgreen;
```
Comment (andresmoschini) [inline assets/scss/modules/_forms.scss]: Why do we need a special hover color for a particular link? (Probably the same question applies for most of the `.forgot-link`'s properties)

File: assets/scss/templates/_signup.scss [hunk 2]
```diff
@@ -6,9 +6,9 @@
   align-items: stretch;
 
   .main-panel {
-    width: 500px;
+    width: 525px;
     height: 100vh;
-    padding: 40px 50px;
+    padding: 40px $dp-spaces--lv6;
```
Comment (andresmoschini) [inline assets/scss/templates/_signup.scss]: 40px is not standard? maybe we should normalize it.

**Comment by andresmoschini** [comment]
Nice, very simple changes!

File: assets/scss/modules/_forms.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/modules/_forms.scss]: All links are green and dark green in hover.
Maybe we would have to change it in base.scss for all the links, and only add the classes for special cases.

File: assets/scss/templates/_signup.scss (hunk 2 above)
Comment (GusBaamonde) [reply assets/scss/templates/_signup.scss]: I'm going to normalize it in another PR

Comment (andresmoschini) [reply assets/scss/templates/_signup.scss]: 👍

**Pull request merged**